### PR TITLE
Feature/#006/add searchbar

### PR DESCRIPTION
**Pull request by CyberGalactus**
search bar #006 please review my code

### CODE REVIEW

File: src/components/searchbar/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import "../searchbar/index.css";
+
+const SearchBar = () => {
+
+
+  return (
+    <div className="cont">
+      <form className="form">
+        <input type="text" className="form-input" name="search" />
+
```
Comment (DavidDyberg) [inline src/components/searchbar/SearchBar.js]: Remove empty row

File: src/components/searchbar/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "../searchbar/index.css";
+
+const SearchBar = () => {
+
+
```
Comment (DavidDyberg) [inline src/components/searchbar/SearchBar.js]: Remove empty row

**Review by DavidDyberg — CHANGES_REQUESTED** [review]
Remove row 6 and 11 in src/components/searchbar/SearchBar.js

File: src/components/searchbar/SearchBar.js (hunk 2 above)
Comment (CyberGalactus) [reply src/components/searchbar/SearchBar.js]: solved

**Review by CyberGalactus — COMMENTED** [review]
solved

**Review by DavidDyberg — APPROVED** [review]
Great!

**Pull request merged**